lib/google_maps_geocoder/google_maps_geocoder.rb

```diff
@@ -96,11 +97,28 @@ def http(uri)
 
   def json_from_url(url)
     uri = URI.parse query_url(url)
+
     logger.debug('GoogleMapsGeocoder') { uri }
+
     response = http(uri).request(Net::HTTP::Get.new(uri.request_uri))
     ActiveSupport::JSON.decode response.body
   end
 
+  def handle_exception(google_response_json)
+    status = google_response_json['status']
+    # no exception
+    return if status == 'OK'
+
+    message = GoogleMapsGeocoderErrorHandler::GeneralGoecodingError\
```

Comment:
Actually, let's stick to Ruby's conventions according to http://blog.honeybadger.io/ruby-exception-vs-standarderror-whats-the-difference/.

TL;DR: Call it `GoogleMapsGeocoder::GoecodingError`, have it extend `StandardError`, and, if necessary, rescue either class—*not* `Exception`.